util/Sequencer/Sequencer.gd

```diff
@@ -1,15 +1,18 @@
-class_name Sequencer extends Node
+extends Node
+
+onready var instruments = $Instruments
+onready var player = $AnimationPlayer
 
 # Data
 var data: SongSequence
 var playing: bool
 var paused: bool
-var player: AnimationPlayer
-
 
 # Functions
 func sequence(sequence: SongSequence):
-	for i in get_children(): i.queue_free()
+	for i in instruments.get_children():
+		i.queue_free()
+
 	data = sequence
 	var song = Animation.new()
 	song.set_step(0.001)
```

Comment:
the add_child should be instruments.add_child

Reply:
```
			song.track_set_path(track_index, inst.name)
```
in the track_set_path as the childs are in instruments the second parameter should be "Instruments"+inst.name

Reply:
Yup, missed that one yesterday, thanks!